### PR TITLE
rerun: init at 0.13.0

### PR DESCRIPTION
**Pull request by SomeoneSerge**
- [x] binaries
- [x] python bindings
- [ ] C++ bindings and CMake config

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/re/rerun/package.nix
Comment (SomeoneSerge) [inline pkgs/by-name/re/rerun/package.nix:1]:
```
[2024-02-16T19:45:03Z INFO  xkbcommon_dl] Failed loading `libxkbcommon.so.0`. Error: CantOpen(DlOpen { desc: "libxkbcommon.so.0: cannot open shared object file: No such file or directory" })
[2024-02-16T19:45:03Z INFO  xkbcommon_dl] Failed loading `libxkbcommon.so`. Error: CantOpen(DlOpen { desc: "libxkbcommon.so: cannot open shared object file: No such file or directory" })
```


Ah yes, these stupid rust packages that don't know to do `DT_NEEDED` and rely on `dlopen` instead...

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 289150` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python311Packages.rerun-sdk</li>
    <li>python311Packages.rerun-sdk.dist</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>rerun</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
The python module fails at the very end with:
```
Running phase: addBuildInputRunpathsPhase
patchelf: not an ELF executable
```

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 289150` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python311Packages.rerun-sdk</li>
    <li>python311Packages.rerun-sdk.dist</li>
    <li>rerun</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
The GUI runs fine on my `x86_64-linux` system.

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 289150` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python311Packages.rerun-sdk</li>
    <li>python311Packages.rerun-sdk.dist</li>
    <li>rerun</li>
  </ul>
</details>

File: pkgs/by-name/re/rerun/package.nix [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        patchelf "$path" --add-rpath "''${_extraRunpaths}"
+      fi
+    done < <(
+      for o in $(getAllOutputNames) ; do
+        find "''${!o}" -type f -and "(" -executable -or -iname '*.so' ")" -print0
+      done
+    )
+
+    unset _extraRunpaths
+    unset _sep
```
Comment (SomeoneSerge) [inline pkgs/by-name/re/rerun/package.nix:101]: Ideally, I'd figure out the minimal set of `--add-needed` required to defuse their `dlopen()`s and avoid enforcing the `--add-rpath`s. I'd rather do this in future PRs...

**Review by GaetanLepage — APPROVED** [review]
LGTM !

**Pull request merged**